### PR TITLE
Bump deconz to 2.18.2

### PR DESCRIPTION
**Pull request by jonas-brauer**

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @jonas-brauer, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;addons#2674).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by jonas-brauer** [comment]
"First-time contributors need a maintainer to approve running workflows."

Hi, I'm a first-time contributor :) 
Any maintainer keen to review this PR? I think it's a fairly quick one

File: deconz/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 6.16.0
+
+- Bump deCONZ to 2.18.2
```
Comment (pfefferle) [inline deconz/CHANGELOG.md:5]: maybe adding a new line here, to be consistent with the rest of the file

Comment (jonas-brauer) [reply deconz/CHANGELOG.md:5]: @pfefferle Thank you, you're correct

**Comment by jonas-brauer** [comment]
@mdegat01 @frenck Is this something for one of you to merge? 😃 (Saw your names on previous deconz PRs)

**Comment by frenck** [comment]
Please, don't ping for reviews. That is not behavior we want in general. Thanks 🙏

**Comment by jonas-brauer** [comment]
All good, I'm new around here and found this one to be a quick win :)
Is there a wiki or documentation of dev guidelines somewhere?

**Comment by jonas-brauer** [comment]
Thank you @pvizeli 🙏
Have a nice weekend!

**Comment by frenck** [comment]
@jonas-brauer There is no schedule or procedure. Just await like everybody else. Everybody thinks their PRs are important or quick wins. With thousands of PRs going through our org, constant pings do not help at all.

You can downvote my comment, which imho is a bit disrespectful, I gave a honest an fair answer in a short notice, even while I am on the road/mobile.

Patience is required in open source projects.

Otherwise, no hard feelings and thanks for contributing ❤️


../Frenck

**Comment by jonas-brauer** [comment]
@frenck Cool 😊
I was actually looking at previous PRs to estimate the pace of the community. PRs like this one usually got attention fairly quickly, so considering this was my first PR here I just wanted to make sure there was nothing I have missed.

It was a longshot pinging the two of you, since you both have merged/reviewed similar ones like this. I did not expect anyone of you to respond unless you wanted to 😊
I'm not respectful against you, as a matter of fact I appreciate you since I just used your addon (https://github.com/hassio-addons/addon-ssh) to solve a problem I had.
However, your answer only told me - Do not make mistakes in this project. (I only got the information of what _not_ to do, not _what_ to do to make this PR happen)

I appreciate you taking the time to answer, but I'd appreciate you even more if you didn't.
A wise mentor of mine told me - You can always tell people "No" later

No hard feelings here either. I learnt something so thank you for all your work and exchanging thoughts on this 👏

**Pull request merged**